lib/generate.sh

```diff
@@ -99,6 +99,32 @@ function _create_sysupdate() {
 }
 # --
 
+function _generate_signed_sysext() {
+  BASEDIR="$1"
+  OUTPUT_IMAGE="$2"
+  FS_FORMAT="$3"
+
+  # Create temporary working directory
+  WORKDIR=$(mktemp -d)
+  trap 'rm -rf "$WORKDIR"' RETURN
+
+  cp -r "/usr/lib/systemd/repart/definitions/sysext.repart.d/" "$WORKDIR"
```

Comment:
Maybe we can keep a know-good copy of these files in a subfolder instead of depending on the host contents